elf_size_analyze/misc.py

```diff
@@ -0,0 +1,20 @@
+"""
+Miscellaneous helper functions
+"""
+
+# construct python regex named group
+def g(name, regex):
```

Comment:
Now when it's in separate file the name could be more verbose (e.g. `named_group()`) and instead use shorthand when importing in other files, e.g.
```python
from elf_size_analyze.misc import named_group as g
```